client/api/omni/specs/omni.proto

```diff
@@ -18,10 +18,12 @@ message MachineSpec {
   reserved 4;
 }
 
-// SecureBootStatus describes the status of the SecureBoot feature.
-message SecureBootStatus {
-  // Enabled is true if SecureBoot is detected to be available and enabled.
-  bool enabled = 1;
+// BootType describes the UKI and secure boot information of the machine.
+message BootType {
+  // IsSecureBoot is true if secure boot is detected to be available and enabled.
+  bool is_secure_boot = 1; // use existing logic
+  // IsFromUki is true if the machine is booted from UKI. Always true if secure boot is enabled.
+  bool is_from_uki = 2;
```

Comment:
It's `BootedWithUKI` in Talos resources, lets make name it the same way here for consistency?

Reply:
Sure, will do. Btw, what do you think about storing the whole SecurityState resource spec as-is?

Reply:
Oh, nvm, probably we don't want to wake up on other fields' changes.